ossfuzz/aspell_fuzzer_corpus/en_us_input

```diff
@@ -0,0 +1,7 @@
+lang=en_us
+encoding=iso-8859-1
+sug-mode=ultra
+mode=none
+ignore=3
+
+This is a test file!
```

Comment:
This file is missing a newline at the end.

This is likely something that I would automatically get fixed (as soon as I open the file in an editor) at some point and just add noise so best to do it now.